src/Tasks/ResolveSDKReference.cs

```diff
@@ -25,6 +25,14 @@ public class ResolveSDKReference : TaskExtension
     {
         #region fields
 
+        /// <summary>
+        /// Platform aliases
+        /// </summary>
+        private static readonly Dictionary<string, string> PlatformAliases = new(StringComparer.OrdinalIgnoreCase)
+        {
+            { "UAP", "windows" }
```

Comment:
```suggestion
            { "UAP", "Windows" }
```

Reply:
Although I'll admit that looks nicer, lower-case windows is what the string actually is, so this adds a layer of complexity to the check I'd like to avoid.

Reply:
No, The `TargetPlatformIdentifier` is `Windows`, even in Windows XAML targets and in the Windows SDK targets. I can confirm. And besides, you're ignoring case, so it doesn't really matter.